backend/backend.py

```diff
@@ -79,13 +94,23 @@ def on_loop(self) -> None:
         if time() > self.prev_recording_status_time + self.recording_status_interval:
             self.send_recording_status()
 
+        if time() > self.prev_camera_is_online_time + self.camera_is_online_interval:
+            self.send_camera_is_online()
+
     @abstractmethod
     def _on_loop(self) -> None:
-        """ Implemented by overlays to perform any neccessary operations that
+        """ Implemented by overlays to perform any necessary operations that
```

Comment:
I've *always* struggled with the spelling of this word lol

Reply:
I have a spellcheck extension for VSCode 😎

Reply:
Haha I actually just got one maybe a week ago. Should have done it ages ago!